src/widgets/PieChart.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2018
+ */
+
+import React, { useState, useEffect } from 'react';
+import { View, StyleSheet } from 'react-native';
+import PropTypes from 'prop-types';
+import { VictoryLabel, VictoryPie } from 'victory-native';
+import { APP_FONT_FAMILY, GREY } from '../globalStyles';
+
+/**
+ * A charting widget for displaying Pie chart report.
+ *
+ * @prop  {string}       id      The report ID.
+ * @prop  {string}       title   The title of the report.
+ * @prop  {string}       type    The type of chart to use to display the report,
+ *                               options are BarChart, LineChart and PieChart.
+ * @prop  {data}         array   An array of {x, y} datapoints to plot.
+ * @prop  {width}        number  The width of the parent container.
+ * @prop  {height}       number  The height of the parent continer.
+ */
+
+export const ReportChart = ({ report: initialReport }, props) => {
```

Comment:
this is two parameters, not one. I suggest just destructuring all:

`export const ReportChart = ({ report: initialReport, title, type, data, width, height}) => { .. }`

Then, in the rest of the component you don't need to use `props.title` etc.. just `title`

Reply:
Also I think you should call this `PieChart`

Reply:
Thanks. In fact, what we are getting is just title, type and data as props, so `export const PieChart = ({ title, type, data }) => { .. }` may be enough. 
Done here 0f3f2a7